packages/react-ai/src/components/AIConversation/types.ts

```diff
@@ -54,6 +60,11 @@ export interface AIConversation {
 
 export type MessageVariant = 'bubble' | 'default';
 
+export interface MessageRenderer {
+  text?: (input: { text: TextContent }) => React.JSX.Element;
```

Comment:
Kinda nit: Maybe give `TextContent` a different name to avoid it conflicting with the `TextContent` component (or vice versa)